.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "typescript.tsdk": "node_modules/.pnpm/typescript@4.9.5/node_modules/typescript/lib",
+  "typescript.enablePromptUseWorkspaceTsdk": true
+}
```

Comment:
The configuration of your IDE should not be in the project 👉🏼 `.gitignore`

Reply:
Done 👍